src/tests/integration/accessTokens.test.ts

```diff
@@ -0,0 +1,27 @@
+import { startAppServer } from '../../server/server'
+import uuidv4 from 'uuid/v4'
+import { accessTokensAdapter } from '../helpers/accessTokensAdapter'
+
+const TEST_PORT = process.env.PORT || 50605
+const ADMIN = process.env.ADMIN || 'hjhj'
+
+describe('/api/accessTokens', () => {
+  let server
+
+  beforeAll(async () => {
+    server = await startAppServer(TEST_PORT, ADMIN)
+  })
+
+  afterAll(() => {
+    server.close()
+  })
+
+  it('checks to see if accesstoken is created', async () => {
+    const name = 'c0d3access'
```

Comment:
you should probably add UUID in here, because we should not be able to create multiple access tokens with the same name. Please also check name in `POST` request to create an accesstoken, make sure you don't create duplicates.

Reply:
Add another test to check for duplicates names.